test/brod_group_coordinator_SUITE.erl

```diff
@@ -0,0 +1,127 @@
+%%%
+%%%   Copyright (c) 2015-2018 Klarna Bank AB (publ)
+%%%
+%%%   Licensed under the Apache License, Version 2.0 (the "License");
+%%%   you may not use this file except in compliance with the License.
+%%%   You may obtain a copy of the License at
+%%%
+%%%       http://www.apache.org/licenses/LICENSE-2.0
+%%%
+%%%   Unless required by applicable law or agreed to in writing, software
+%%%   distributed under the License is distributed on an "AS IS" BASIS,
+%%%   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%%%   See the License for the specific language governing permissions and
+%%%   limitations under the License.
+%%%
+
+%% @private
+-module(brod_group_coordinator_SUITE).
+-define(CLIENT_ID, ?MODULE).
+-define(OTHER_CLIENT_ID, other_coordinator_id).
+-define(TOPIC, <<"brod-group-subscriber-1">>).
+-define(GROUP, <<"brod-group-coordinator">>).
+-define(PARTITION, 0).
+
+%% Test framework
+-export([ init_per_suite/1
+        , end_per_suite/1
+        , common_init_per_testcase/2
+        , common_end_per_testcase/2
+        , suite/0
+        ]).
+
+%% brod coordinator callbacks
+-export([ assignments_revoked/1
+        , assignments_received/4
+        ]).
+
+%% Test cases
+-export([ t_acks_during_revoke/1 ]).
+
+-define(assert_receive(Pattern, Return),
+  receive
+    Pattern -> Return
+  after
+    30000 -> ct:fail(erlang:process_info(self(), messages))
+  end).
+
+-include_lib("snabbkaffe/include/ct_boilerplate.hrl").
+-include("brod.hrl").
+
+%%%_* ct callbacks =============================================================
+
+suite() -> [{timetrap, {seconds, 60}}].
+
+init_per_suite(Config) ->
+  {ok, _} = application:ensure_all_started(brod),
```

Comment:
Better move it to `common_init_per_testcase` (and stop brod app in `common_end_per_testcase`). Brod has tendency to start processes under its own supervisor, which may lead to lingering children. Even though this is not relevant in this testsuite yet, it's always good to keep things clean from the very beginning